components/identity-core/org.wso2.carbon.identity.base/src/main/java/org/wso2/carbon/identity/base/IdentityConstants.java

```diff
@@ -513,4 +513,16 @@ private IdentityConstants() {
         public static final String MAX_AGE = "CORS.MaxAge";
         public static final String TAG_REQUESTS = "CORS.TagRequests";
     }
+
+    /**
+     * Contains the constants related to Legacy Feature config elements.
+     */
+    public static class LegacyFeatureConfigElements {
```

Comment:
let;s make the constructor of this class private

Reply:
Why do we need to make this constructor private? I need to access this constructor from out of this package.